packages/nodes-base/nodes/Form/Form.node.ts

```diff
@@ -105,7 +107,7 @@ const completionProperties = updateDisplayOptions(
 			options: [
 				{
 					name: 'Show Completion Screen',
-					value: 'text',
+					value: 'completionScreen',
```

Comment:
this should be under new version, will break workflows that have completion screen